recipes-tag/h5py/run_test.py

```diff
@@ -19,7 +19,6 @@
 import h5py.h5t
 import h5py.h5z
 import h5py.utils
-assert h5py.version.hdf5_version_tuple == (1, 10, 1)
```

Comment:
Do you know why it's deleted? Was it our channel's modification to check for the strict hdf5 version?

Reply:
I run some tests, so hdf5 version can be 1.10.0 or 1.10.2 or higher. I didn't see restriction on this. 

I just copied from conda-forge, and didn't make any change.

Reply:
OK, sounds good.